pages/home/passkeys-tutorials/safe-passkeys-tutorial.mdx

```diff
@@ -182,35 +201,44 @@ h3 {
 
 ## Testing your Safe passkeys app
 
 That's it! You can find the source code for the example created in this tutorial [on GitHub](https://github.com/5afe/safe-passkeys-tutorial). You can now return to your browser and see the app displayed 🎉.
 
 ![safe-passkeys-app-1.png](../../../assets/safe-passkeys-app-1.png)
 
-Click the **Add New Passkey** button to prompt a browser pop-up asking you to confirm the creation of a new passkey. This passkey will be stored in your browser's local storage and displayed in the list above the button.
+Click the **Create a New Passkey** button to prompt a browser pop-up asking you to confirm the creation of a new passkey. This passkey will be stored in your browser's local storage and displayed in the list above the button.
 
-Once confirmed, select this passkey by clicking **Select** next to it. This will prompt another pop-up window, this time asking to confirm the use of the previously created passkey.
+This will create a new Safe object in the background, which will be deployed when you click the **Mint NFT** button. This will also mint an NFT using the passkey you created.
 
-![safe-passkeys-app-2.png](../../../assets/safe-passkeys-app-2.png)
+![safe-passkeys-app-3.png](../../../assets/safe-passkeys-app-3.png)
 
-At this stage, the app will have created a safe object awaiting deployment. By clicking **Sign transaction with passkey**, the deployment of this Safe will then be embedded in a batch transaction, along with the minting of the NFT.
-Open the console to see the UserOp that was sent or click the link provided to Jiffy scan for more complete information.
+Click the link to Jiffy Scan to see the UserOp that was sent and more complete information.
 
 ## Do more with Safe and passkeys
 
 We learned how to use passkeys (create them, store them, and use them securely) and how they can interact with a Safe (deploy it and send transactions). We hope you enjoyed this tutorial and that the combination of passkeys and the ERC-4337 will unlock new forms of ownership for your project and users.
 
-You can now integrate passkeys with more transactions and functionalities of the Safe ecosystem. You can read more about passkeys in our [overview](../passkeys-overview.mdx) or in the [WebAuthn API documentation](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API).
+You can read more about passkeys in our [overview](../passkeys-overview.mdx) or in the [WebAuthn API documentation](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API).
+
+<Callout type='warning' emoji='⚠️'>
```

Comment:
Should this be a sub-section called Best practices or something similar instead of a callout?